src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,25 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
```

Comment:
O Joi poderia já receber esse parâmetro como número inteiro, e também poderia ter um valor padrão caso o usuário não fornecesse o mesmo.

```suggestion
            offset: Joi
                .number()
                .integer()
                .min(0)
                .default(0),
```